### DISCUSSION TITLE
Clarification on 1-bit Adam speed comparison for SQuAD

### DISCUSSION THREAD

**Discussion by sarvghotra** [deepspeedai/DeepSpeed#1648] [Q&A]
Hi, it seems counter-intuitive to me that Infiband scales worse than TCP as shown in the following figure from DeepSpeed [wiki here](https://www.deepspeed.ai/news/2020/09/08/onebit-adam-blog-post.html#3-benefits-of-1-bit-adam-on-communication-constrained-systems):

![image](https://user-images.githubusercontent.com/9509853/146625898-729b3b9c-c539-426e-a0f6-9ee978fdd04e.png)

### **So wanted to confirm if the figure is correct.**

**Comment by conglongli** [comment]
Need confirmation from @awan-10 , but I believe this is a typo that TCP lines are actually Infinibands, and Infiniband lines are actually TCP. On the other hand, for most up to date results please refer our final paper in ICML http://proceedings.mlr.press/v139/tang21a/tang21a.pdf
